cfme/tests/infrastructure/test_host.py

```diff
@@ -541,28 +540,29 @@ def test_add_ipmi_refresh(appliance, setup_provider):
 
 
 @test_requirements.infra_hosts
-@pytest.mark.parametrize("num_hosts", [1, 2, 4])
-@pytest.mark.meta(blockers=[BZ(1634794, forced_streams=["5.10"])], automates=[1634794])
-def test_infrastructure_hosts_crud(appliance, setup_provider_min_hosts, provider, num_hosts):
+@pytest.mark.parametrize("crud_action", ['edit_from_hosts', 'edit_from_details', 'remove'])
+def test_infrastructure_hosts_crud(appliance, setup_provider, provider, crud_action):
     """
     Polarion:
         assignee: prichard
         casecomponent: Infra
         caseimportance: low
         initialEstimate: 1/6h
-    Bugzilla:
-        1634794
     """
-    my_slice = slice(0, num_hosts, None)
-    hosts_view = navigate_to(provider.collections.hosts, "All")
-    for h in hosts_view.entities.get_all(slice=my_slice):
-        h.ensure_checked()
-    hosts_view.toolbar.configuration.item_select('Edit Selected items',
-                                                handle_alert=False)
-    try:
-        hosts_view.navigation.select('Compute', 'Infrastructure', 'Providers', handle_alert=False)
-        final_view = provider.create_view(InfraProvidersView)
-        assert final_view.is_displayed
-    except UnexpectedAlertPresentException:
-        pytest.fail("Abandon changes alert displayed, but no changes made.")
-    # TODO add additional crud functionality.
+    from cfme.utils.update import update
```

Comment:
can you move this to the top of the file?

Reply:
Done